cli/cli.go

```diff
@@ -50,9 +50,6 @@ Output build version information.
 
 var cockroachCmd = &cobra.Command{
 	Use: "cockroach",
-	PersistentPreRun: func(cmd *cobra.Command, args []string) {
-		context.Addr = util.EnsureHostPort(context.Addr)
-	},
```

Comment:
I'm separating addr and port in
https://github.com/cockroachdb/cockroach/pull/3757
Part of that change involves proper defaults, and always concatenating host
and port.
On Jan 19, 2016 8:07 AM, "Peter Mattis" <notifications@github.com> wrote:

> In cli/cli.go
> <https://github.com/cockroachdb/cockroach/pull/3911#discussion_r50104446>:
>
> > @@ -50,9 +50,6 @@ Output build version information.
> >
> >  var cockroachCmd = &cobra.Command{
> >  	Use: "cockroach",
> > -	PersistentPreRun: func(cmd *cobra.Command, args []string) {
> > -		context.Addr = util.EnsureHostPort(context.Addr)
> > -	},
>
> Ah, this is where we ensure a port on context.Addr. I'm curious who you
> didn't just add a line here to ensure a port on context.PGAddr. You'd
> have to change EnsureHostPort to take a port to append if one isn't
> present. As it is, there is a lot of duplicated functionality between
> EnsureHostPort and your new portPadding function.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/3911/files#r50104446>.
>